source/components/index.rst

```diff
@@ -20,14 +20,14 @@ Creating components
          .decoration(TextDecoration.BOLD, true)
      )
      .append(Component.text(" to jump!"));
-   // now you can send `textComponent` to something, such as a client
+   // Now you can send `textComponent` to something, such as a client.
 
 You can also use a builder, which is mutable, and creates one final
 component with the children.
 
 .. code:: java
 
-   // Creates a line of text saying "You're a Bunny! Press <key> to jump!", with some colouring and styling.
+   // Creates a line of text saying "You're a Bunny! Press <key> to jump!" with colouring and styling.
```

Comment:
why? doesn't seem like a necessary change

Reply:
The comma wasn't needed and removing "some" should shorten the line enough so that a horizontal scrolling bar won't appear in most browsers. But yes, this wasn't necessary.